lib/transform.js

```diff
@@ -168,3 +169,23 @@ function mapCheckItemIssueToMkdn(issue) {
   }
   return `- [${checked}] #${issue.number}`;
 }
+
+Transform.usernameToId = function (username) {
+  if (!username) {
+    return;
+  } else {
```

Comment:
Return early and de-nest. There's no need to have a big else block when you don't need one at all because that condition is completely handled.

https://www.youtube.com/watch?v=5DVV36uqQ4E